wallet-new/src/Cardano/Wallet/API/Dev/Test.hs

```diff
@@ -0,0 +1,10 @@
+module Cardano.Wallet.API.Dev.Test where
+
+import           Servant.API.ContentTypes (OctetStream)
+import           Cardano.Wallet.API.Response (WalletResponse)
+import           Pos.Wallet.Web.Methods.Misc (WalletStateSnapshot)
+
+import           Servant
+
+type API = "dump-wallet-state" :> Summary "Dump wallet state."
+                               :> Get '[OctetStream] (WalletResponse WalletStateSnapshot)
```

Comment:
I thought this returned a JSON -- isn't this the case? I _think_ (but I might be wrong) that simply returning a JSON is nicer and more development friendly than dumping a file on disk.

Reply:
V0 endpoint returns a `OctetStream` (see https://github.com/input-output-hk/cardano-sl/blob/develop/wallet/src/Pos/Wallet/Web/Api.hs#L153-L156), so I'm not sure why we should change it.

Reply:
> V0 endpoint returns a OctetStream

I suspect because the `WalletStorage` could be huge and they didn't want to display it all as a JSON. But if we go down the development route, we shouldn't really call this in production (where the state stored in the `WalletStorage` can be huge), so I think we can afford the luxury of showing just a nice JSON. What do you think?

Reply:
Let's ask the Daedalus team as an user of this endpoint ^:
@darko-mijic Would your team still happy, if the `V1` endpoint `/api/development/dump-wallet-state` (corresponding `V0` endpoint is `/api/test/state`) will return a `JSON` and not a `OctetStream` we had before?

Reply:
Great idea -- probably we also want to page @DominikGuzei.